.github/workflows/reusable_docker_build.yml

```diff
@@ -23,25 +19,12 @@ jobs:
 
     steps:
       - name: 'Check out the repo'
-        id: checkout
         uses: actions/checkout@v4
 
-      - name: 'Determine files differing from target branch'
-        if: ${{ github.event_name != 'schedule' }}
-        id: changed-files
-        uses: tj-actions/changed-files@v44
-        with:
-          files: |
-            ${{ inputs.changed-files }}
-
       - name: 'Set up Docker buildx'
-        id: buildx
-        if: ${{ steps.changed-files.outputs.any_changed == 'true' || github.event_name == 'schedule' }}
         uses: docker/setup-buildx-action@v3
 
       - name: 'Log into GitHub Container Repository'
-        id: login
```

Comment:
Minor point: any reason you got rid of the `id`s? I guess just because they are superfluous?

Reply:
Yup! We no longer need them. Would you prefer me to bring them back?

Reply:
No that's fine, feel free to remove.